examples/README.md

```diff
@@ -0,0 +1,41 @@
+# PSyclone Examples
+
+**Author:** J. Henrichs, Bureau of Meteorology
+
+# Introduction
+
+This directory contains examples for the various APIs that PSyclone
+supports, and data structures. In detail:
+
+## [gocean](./gocean)
+Various examples related to the GOCean API.
+
+## [nemo](./nemo)
+Various examples related to the NEMO API.
+
+## [lfric](./lfric)
+Various examples related to the LFric API.
+
+## [line_length](./line_length)
+This example demonstrates the use of the line length option.
+
+## [psyir](./psyir)
+This directory contains examples on how to use PSyIR to create
+Fortran and C code as output.
+
+## [stub_generation](./stub_generation)
+This directory shows the usage of the stub-generation utility
+included in PSyclone, which helps generating a Kernel's argument
+list and its datatypes from the Kernel metadata.
+
+## transformations
```

Comment:
Yes, but:
1) transformation has no README.md that I could link to
2) I have no idea why this one example is here and not in the LFRIc dir ... though that could be because
3) it is  actually still dynamo0.1

In order to progress this PR, I have opened #1100 for this.

Reply:
Thank you!